.coveragerc

```diff
@@ -0,0 +1,8 @@
+[run]
+omit =  
+ *__init__*
+ *authors/apps/authentication/migrations/*     
+ *authors/apps/authentication/tests_/*
+ *authors/settings.py*
```

Comment:
@MuhweziDeo there is no need of omitting `settings.py` and `url.py` since they are always covered 100%.
the rest of the files such as `tests` and `migrations` can be ignored in the `.travis.yml` file when running the tests by passing the `--omit` flag to the test command. In other words, we may not need this `.coveragerc` file.

Reply:
should i remove them @ja-odur

Reply:
@MuhweziDeo yes